public/js/components/SourceTabs.js

```diff
@@ -139,6 +140,13 @@ const SourceTabs = React.createClass({
       CloseButton({ handleClick: onClickClose }));
   },
 
+  renderNewButton() {
+    return dom.div({
+      className: "new-tab-btn",
+      onClick: () => this.props.openFileSearch(true)
```

Comment:
so this wasn't clear because of the dark theme tabs, but is it possible to have multiple blank tabs?

It would be nice if the tabs button were hidden during file search.